cmd/controller/sparsefilegenerator.go

```diff
@@ -161,6 +164,23 @@ func GetSparseDiskUuid(hostname string, sparseFile string) string {
 	return SPARSE_DISK_PREFIX + util.Hash(hostname+sparseFile)
 }
 
+func GetActiveSparseDisksUuids(hostname string) []string {
+	sparseFileLocation := GetSparseFileDir()
+	sparseUids := make([]string, 0)
+	files, err := ioutil.ReadDir(sparseFileLocation)
+	if err != nil {
+		glog.Error("Failed to read sperse file names : ", err)
+		return sparseUids
+	}
+	for _, file := range files {
+		if strings.HasSuffix(file.Name(), SPARSE_FILE_NAME) {
+			fileName := path.Join(GetSparseFileDir(), file.Name())
```

Comment:
@shovanmaity You don't need to call `GetSparseFileDir` again as you have already did that in the first line of this function.

Reply:
done :+1: